### DISCUSSION TITLE
How to get attribute numbers not listed in categories.yaml

### DISCUSSION THREAD

**Discussion by 0verEngineer** [Second-Hand-Friends/kleinanzeigen-bot#336] [Q&A] [answered]
I want to list some ads with categories not listed in the categories.yaml, how to find out which numbers i need to use?

**Comment by 0verEngineer** [comment, marked as answer]
Ok i found it out, it is quite easy:

1. Go to https://www.kleinanzeigen.de/s-kategorien.html
2. Click on one of the base categories like "[Auto, Rad & Boot](https://www.kleinanzeigen.de/s-auto-rad-boot/c210)"
3. Check the url it has the number of the category at the end: "c210", remove the "c"
4. Go to the subcategory like [Autos](https://www.kleinanzeigen.de/s-autos/c216)
5. Check the url again and remove the "c" from the number
6. Now you have 210/216
7. Click on the "Art/Marke" like "Volkswagen" and check the url, the last word of the url is the art, in this case it is "volkswagen"
8. Now you have 210/216/volkswagen

**Comment by 0verEngineer** [comment]
It would be good to add this to the README

**Reply by sebthom** [reply]
A PR is welcome.
